src/main/java/studio/magemonkey/genesis/managers/item/ItemDataPart.java

```diff
@@ -162,7 +163,7 @@ public static List<String> readItem(ItemStack item) {
         return output;
     }
 
-    public static boolean isSimilar(ItemStack shopItem,
+    public static boolean isMetaSimilar(ItemStack shopItem,
```

Comment:
This shouldn't be renamed -- it is used in ItemStackChecker to check for price types and things